nym-vpn-apple/Scripts/BuildCore.sh

```
@@ -34,6 +34,27 @@ echo "[BuildCore] CORE_ROOT=${CORE_ROOT}"
+# (The compile-time #if SANTA is handled by Package.swift: debug config defines
+# it locally, ship Release-without-NYM_SANTA strips it.)
+DAEMON_PLIST="${APPLE_ROOT}/NymVPNDaemon/Resources/Info.plist"
+if [[ -f "${DAEMON_PLIST}" ]]; then
+  if [[ "${RELEASE}" == "false" ]]; then
+    # String, not bool: isRunningOnCI reads this via `as? String` (a bool cast
+    # fails → Santa silently off). Tracked plist + CI Set both keep it a string.
+    /usr/libexec/PlistBuddy -c "Set :IsCiBuild true" "${DAEMON_PLIST}" 2>/dev/null \
+      || /usr/libexec/PlistBuddy -c "Add :IsCiBuild string true" "${DAEMON_PLIST}"
+    echo "[BuildCore] 🎅 Santa's menu enabled (IsCiBuild=true)"
+  else
+    /usr/libexec/PlistBuddy -c 'Set :IsCiBuild $(IS_CI_BUILD)' "${DAEMON_PLIST}" 2>/dev/null || true
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)